.cron.yml

```diff
@@ -10,3 +10,9 @@ jobs:
           target-tasks-method: l10n_screenshots
           treeherder-symbol: l10-screenshots
       when: []  # Manual trigger only
+    - name: tab-performance-test
+      job:
+          type: decision-task
+          target-tasks-method: tab_performance_test
+          treeherder-symbol: tab-performance-test
```

Comment:
@JohanLorenzo would this symbol be fine or do we need an existing one?

Reply:
Fine by me 👍